ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/CompletionInfo.scala

```diff
@@ -4,26 +4,27 @@
+  * Unless noted otherwise, the data is the same as for [[CompletionInfo]].
   *
-  * @param actAs                  the non-empty set of parties that submitted the change.
+  * @param actAs                   the non-empty set of parties that submitted the change.
   * @param applicationId          an identifier for the Daml application that submitted the command.
   * @param commandId              a submitter-provided identifier to identify an intended ledger change
   *                               within all the submissions by the same parties and application.
   * @param optDeduplicationPeriod The deduplication period that the [[WriteService]] actually uses for the command submission.
-  *                               It may differ from the suggested deduplication period given to [[WriteService.submitTransaction]] or [[WriteService.rejectSubmission]].
-  *                               For example, the suggested deduplication period may have been converted into a different kind or extended.
-  *                               The particular choice depends on the particular implementation.
-  *                               This allows auditing the deduplication guarantee described in the [[ReadService.stateUpdates]].
+  *                                It may differ from the suggested deduplication period given to [[WriteService.submitTransaction]] or [[WriteService.rejectSubmission]].
+  *                                For example, the suggested deduplication period may have been converted into a different kind or extended.
+  *                                The particular choice depends on the particular implementation.
+  *                                This allows auditing the deduplication guarantee described in the [[ReadService.stateUpdates]].
   *
-  *                               Optional as some implementations may not be able to provide this deduplication information.
-  *                               If an implementation does not provide this deduplication information,
-  *                               it MUST adhere to the deduplication guarantee under a sensible interpretation
-  *                               of the corresponding [[SubmitterInfo.deduplicationPeriod]].
-  * @param submissionId An identifier for the submission that allows an application
-  *   to correlate completions to its submissions.
+  *                                Optional as some implementations may not be able to provide this deduplication information.
+  *                                If an implementation does not provide this deduplication information,
+  *                                it MUST adhere to the deduplication guarantee under a sensible interpretation
+  *                                of the corresponding [[CompletionInfo.deduplicationPeriod]].
+  * @param submissionId            An identifier for the submission that allows an application
+  *                               to correlate completions to its submissions.
```

Comment:
I am not sure I understand these ScalaDoc indentation changes

Reply:
IntelliJ has made it happen. I actually hadn't realized it did it, it must have been on commit. Shame on me for not reading through my PR before opening it.

I will make sure they're all aligned.

Reply:
Re-aligned all logs in _domain.scala_, _CompletionInfo.scala_, and _Update.scala_.